maxim/logger/openai/utils.py

```
@@ -166,33 +146,58 @@ def parse_completion_from_chunks(
+        # Construct the final response
+        response = {
+            "id": last_chunk.id,
+            "object": "chat.completion",
+            "created": int(time.time()),
+            "model": getattr(last_chunk, "model", None),
+            "choices": [{
+                "index": 0,
+                "message": {
+                    "role": "assistant",
+                    "content": combined_content if not tool_calls else None,
+                    "tool_calls": tool_calls if tool_calls else None
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)